app-crypt/libsecret/libsecret-0.21.0.ebuild

```diff
@@ -12,7 +12,7 @@ HOMEPAGE="https://wiki.gnome.org/Projects/Libsecret"
 LICENSE="LGPL-2.1+ Apache-2.0" # Apache-2.0 license is used for tests only
 SLOT="0"
 
-IUSE="+crypt gtk-doc +introspection test test-rust tpm +vala"
+IUSE="+crypt gtk-doc +introspection man test test-rust tpm +vala"
```

Comment:
This needs justification in the commit message and probably isn't appropriate in this case.

Please see https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0305.

If the man page is already in the dist tarball and the build system option here is to _regenerate_ it, then we can make it conditional (or just disable it) though.

Reply:
* It's okay to have a USE if it's to _regenerate_ the man pages as opposed to _generate_ them, but in such cases, you may want to just disable it entirely. That is fine as long as man pages are still installed (no need to regenerate them if they're in the tarball, is the point).

* texinfo should probably be treated as equivalent to man pages.

* So, it's fine to disable man page building if the tarball already had `man/foo.1` in there and it gets installed and if the option is purely to regenerate it from `man/foo.md` or something.

* It's fine for HTML documentation to be guarded with a USE flag, unless it's small and has no dependencies, in which case it's probably worth just installing it (see https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301 as well).

* handbook should be treated like doc. Both are fine to keep conditional in general as they usually require extra dependencies and often aren't included in the dist tarballs (pre-generated) either.

Reply:
Regeneration is where there's one in the tarball (already exists) but we're building a new one.
Generation is where there's no existing one and we have to make one.

Reply:
Good question. I'd probably say we should keep the additions.